src/commands/dev/lib/dev-builder.ts

```diff
@@ -58,6 +60,26 @@ export async function executeInitialBuilds(
   }
 }
 
+export async function executePrepareCache(
+  buildConfig: BuildConfig,
+  params: PrepareCacheParams
+): Promise<CacheOutputs> {
+  const { builder } = buildConfig;
+  if (!builder) {
+    throw new Error('No builder');
+  }
+  if (!builder.prepareCache) {
+    throw new Error('Builder has no `prepareCache()` function');
+  }
+
+  // Since the `prepareCache()` function may be computationally expensive, and
+  // its run in the same process as `now dev` (for now), defer executing it
+  // until after there has been time for the current HTTP request to complete.
+  await new Promise(r => setTimeout(r, 3000));
```

Comment:
Is there a way to listen for the HTTP req to finish rather than waiting an arbitrary number?